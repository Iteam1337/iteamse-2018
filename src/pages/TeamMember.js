// @flow

import * as React from 'react'
import * as Iteam from '../typings/iteam.flow'
import * as IteamCMS from './__generated__/TeamMemberPage'
import Team from '../components/Team/Team'
import Header from '../components/Header/Header'
import Block from '../components/Blocks/Block'
import Breadcrumbs from '../components/Breadcrumbs/Breadcrumbs'
import { Query } from 'react-apollo'
import gql from 'graphql-tag'
import GridColumn from '../components/Grid/GridColumn'
import PhoneNumber from '../components/Link/PhoneNumber'
import UnstyledList from '../components/List/UnstyledList'
import Mailto from '../components/Link/Mailto'

type Props = {
  match: {
    params: {
      shortName: string,
    },
  },
}

type QueryProps = Iteam.ApolloBase<IteamCMS.TeamMemberPage>

export const TeamMemberPageQuery = gql`
  query TeamMemberPage($shortName: String!) {
    teamMember(shortName: $shortName) {
      avatar
      background
      backgroundTitle
      competence
      competenceTitle
      email
      headerImage
      headerTextBgColor
      location
      name
      phoneNumber
      title
      why
      whyTitle
    }
  }
`

const TeamMemberPage = ({ match }: Props) => {
  return (
    <Query
      query={TeamMemberPageQuery}
      variables={{ shortName: match.params.shortName }}
    >
      {({ loading, data: { teamMember } }: QueryProps) => {
        if (loading) {
          return null
        }

        return (
          <React.Fragment>
            <Header
              backgroundImage={teamMember.headerImage}
              messageBgColor={teamMember.headerTextBgColor}
              messageOne={teamMember.name}
              messageTwo={teamMember.title}
            />

            <GridColumn>
              <Breadcrumbs title={teamMember.name} />

              <Block title="Kontakt">
                <UnstyledList>
                  {teamMember.phoneNumber && (
                    <li>
                      <PhoneNumber phoneNumber={teamMember.phoneNumber}>
                        {teamMember.phoneNumber}
                      </PhoneNumber>
                    </li>
                  )}
                  <li>
                    <Mailto email={teamMember.email}>{teamMember.email}</Mailto>
                  </li>
                </UnstyledList>
              </Block>

              <Block title={teamMember.whyTitle}>{teamMember.why}</Block>

              <Block title={teamMember.backgroundTitle}>
                {teamMember.background}
              </Block>

              <Block title={teamMember.competenceTitle}>
                {teamMember.competence}
              </Block>
            </GridColumn>

            <Team bgColor="green" shortName={['hrn', 'jmn']} />
          </React.Fragment>
        )
      }}
    </Query>
  )
}

export default TeamMemberPage
