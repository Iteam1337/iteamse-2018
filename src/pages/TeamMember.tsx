import gql from 'graphql-tag'
import * as React from 'react'
import { Query } from 'react-apollo'
import { Helmet } from 'react-helmet'
import { RouteComponentProps, withRouter } from 'react-router-dom'
import { TeamMemberPageQuery } from '../../typings/iteamse'
import Block from '../components/Blocks/Block'
import Breadcrumbs from '../components/Breadcrumbs/Breadcrumbs'
import GridColumn from '../components/Grid/GridColumn'
import Header from '../components/Header/Header'
import Mailto from '../components/Link/Mailto'
import PhoneNumber from '../components/Link/PhoneNumber'
import UnstyledList from '../components/List/UnstyledList'
import Team from '../components/Team/Team'
import styled from '../theme'
import { set } from '../utils/googleAnalytics'

export const TEAM_MEMBER_PAGE_QUERY = gql`
  query TeamMemberPage($shortName: String!) {
    teamMember(shortName: $shortName) {
      avatar
      background
      backgroundTitle
      competence
      competenceTitle
      email
      headerImage
      headerTextBgColor
      location
      name
      phoneNumber
      team
      title
      why
      whyTitle
    }
  }
`

const TeamMember = styled.div`
  h1:first-of-type {
    font-weight: 300;
  }
`

class TeamMemberQuery extends Query<TeamMemberPageQuery> {}

export class TeamMemberPage extends React.Component<
  RouteComponentProps<{ match: string; shortName: string }>
> {
  componentDidMount() {
    set(this.props.location.pathname)
  }

  render() {
    const { match } = this.props
    return (
      <TeamMemberQuery
        query={TEAM_MEMBER_PAGE_QUERY}
        variables={{ shortName: match.params.shortName }}
      >
        {({ loading, data }) => {
          if (loading || !data) {
            return null
          }

          const { teamMember } = data

          if (!teamMember) {
            return <div>Teammedlemmen existerar inte</div>
          }

          return (
            <>
              <Helmet>
                <title>Iteam |Â {teamMember.name}</title>
                <meta name="og:title" content={`Iteam | ${teamMember.name}`} />
                <meta
                  name="twitter:title"
                  content={`Iteam | ${teamMember.name}`}
                />
                {teamMember.headerImage && (
                  <meta
                    name="og:image"
                    content={`https:${teamMember.headerImage}`}
                  />
                )}
              </Helmet>
              <TeamMember>
                <Header
                  backgroundImage={teamMember.headerImage}
                  messageBgColor={teamMember.headerTextBgColor}
                  messageOne={teamMember.name}
                  messageTwo={teamMember.title}
                />

                <GridColumn>
                  <Breadcrumbs title={teamMember.name} />

                  <Block title="Kontakt">
                    <UnstyledList>
                      {teamMember.phoneNumber && (
                        <li>
                          <PhoneNumber phoneNumber={teamMember.phoneNumber}>
                            {teamMember.phoneNumber}
                          </PhoneNumber>
                        </li>
                      )}
                      <li>
                        <Mailto email={teamMember.email}>
                          {teamMember.email}
                        </Mailto>
                      </li>
                    </UnstyledList>
                  </Block>

                  <Block title={teamMember.whyTitle}>{teamMember.why}</Block>

                  <Block title={teamMember.backgroundTitle}>
                    {teamMember.background}
                  </Block>

                  <Block title={teamMember.competenceTitle}>
                    {teamMember.competence}
                  </Block>
                </GridColumn>

                <Team bgColor="green" shortName={teamMember.team} />
              </TeamMember>
            </>
          )
        }}
      </TeamMemberQuery>
    )
  }
}

export default withRouter(TeamMemberPage)
