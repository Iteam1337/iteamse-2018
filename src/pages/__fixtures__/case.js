export const workCase = {
  headerBgColor: '#1616B2',
  logo:
    '//images.ctfassets.net/rj4r6yfcesw5/4p0ptdLj9SaECekOI4c6C2/c2cdeab1cebc9650ebb0e7778357c231/AF_ordbild_vit.png',
  slug: 'arbetsformedlingen',
  thumbnailImage:
    '//images.ctfassets.net/rj4r6yfcesw5/3Y7zZ25WbeOCU8w84kAEos/b97ff1899cf6f2e4dca0eb2b4f0d5a44/af.png',
  tags: ['API', 'Docker', 'UX', 'React', 'Vue'],
  title: 'Arbetsförmedlingen',
  introduction:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  introductionTitle: 'En ny digital tjänst',
  process:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  processTitle: 'Process',
  development:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  developmentTitle: 'Utveckling',
  aboutCompany:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  aboutCompanyTitle: 'Arbetsförmedlingen',
  partners: '[Digitalbyrån](https://hej.com)',
  partnersTitle: 'Samarbetspartners',
  contact:
    '- [Hans Rollman](mailto:hans.rollman@iteam.se)\n- Johan Littorin, Telenor, 0734 41 72 85',
  contactTitle: 'Presskontakt',
  quote:
    '"We feel that our crew at Iteam are more colleagues than suppliers. Rather than specifying a tech spec to work on, we collaborate, carefully chiseling out functionality as we go along. This way of working has minimized waste and it has also taken us further than we ever imagined”',
  quotePerson: 'Anna Andersson, chef Arbetsförmedlingen',
  __typename: 'Case',
}
