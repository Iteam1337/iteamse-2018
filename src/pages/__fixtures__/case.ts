export const workCase = {
  __typename: 'Case',
  aboutCompany:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  aboutCompanyTitle: 'Arbetsförmedlingen',
  contact:
    '- [Hans Rollman](mailto:hans.rollman@iteam.se)\n- Johan Littorin, Telenor, 0734 41 72 85',
  contactTitle: 'Presskontakt',
  development:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  developmentTitle: 'Utveckling',
  headerBgColor: '#1616B2',
  introduction:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  introductionTitle: 'En ny digital tjänst',
  logo:
    '//images.ctfassets.net/rj4r6yfcesw5/4p0ptdLj9SaECekOI4c6C2/c2cdeab1cebc9650ebb0e7778357c231/AF_ordbild_vit.png',
  partners: '[Digitalbyrån](https://hej.com)',
  partnersTitle: 'Samarbetspartners',
  process:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  processTitle: 'Process',
  quote:
    '"We feel that our crew at Iteam are more colleagues than suppliers. Rather than specifying a tech spec to work on, we collaborate, carefully chiseling out functionality as we go along. This way of working has minimized waste and it has also taken us further than we ever imagined”',
  quotePerson: 'Anna Andersson, chef Arbetsförmedlingen',
  slug: 'arbetsformedlingen',
  tags: ['API', 'Docker', 'UX', 'React', 'Vue'],
  team: null,
  thumbnailImage:
    '//images.ctfassets.net/rj4r6yfcesw5/3Y7zZ25WbeOCU8w84kAEos/b97ff1899cf6f2e4dca0eb2b4f0d5a44/af.png',
  title: 'Arbetsförmedlingen',
}
