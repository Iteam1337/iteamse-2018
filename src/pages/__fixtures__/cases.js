export const cases = [
  {
    location: 'Stockholm',
    shortDescription:
      'Synliggörande av asylsökandes kompetens med digitala verktyg',
    slug: 'vimla',
    thumbnailImage:
      '//images.ctfassets.net/rj4r6yfcesw5/68OlBclfQAGMcEUsQgcIUY/9bdf52bb8a9a17be06e11ef3fa32d3de/vimla-case-phone.png',
    title: 'Vimla',
    __typename: 'Case',
  },
  {
    location: 'Stockholm',
    shortDescription:
      'Synliggörande av asylsökandes kompetens med digitala verktyg',
    slug: 'tsab',
    thumbnailImage:
      '//images.ctfassets.net/rj4r6yfcesw5/UGucAeFycSqIGw0CAc0ys/3d58300261bcff64dba751de9228a150/tsab-case-phone.png',
    title: 'Taxi Stockholm',
    __typename: 'Case',
  },
  {
    location: 'Stockholm',
    shortDescription:
      'Synliggörande av asylsökandes kompetens med digitala verktyg',
    slug: 'seb',
    thumbnailImage: null,
    title: 'SEB',
    __typename: 'Case',
  },
  {
    location: 'Stockholm',
    shortDescription:
      'Synliggörande av asylsökandes kompetens med digitala verktyg',
    slug: 'arbetsformedlingen',
    thumbnailImage:
      '//images.ctfassets.net/rj4r6yfcesw5/3Y7zZ25WbeOCU8w84kAEos/b97ff1899cf6f2e4dca0eb2b4f0d5a44/af.png',
    title: 'Arbetsförmedlingen',
    __typename: 'Case',
  },
]
