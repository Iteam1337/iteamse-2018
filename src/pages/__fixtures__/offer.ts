// tslint:disable:object-literal-sort-keys
export const offers = [
  {
    offerIntroImage:
      '//images.ctfassets.net/rj4r6yfcesw5/5n7J4mzx3UWam0YwgsUQm/8e00d12d6e80eba927b2a313fb250d87/1.png',
    offerTitle: 'Kod',
    offerIntroText:
      'Systemutveckling, automatisering, artificiell intelligens, maskininlärning. Vi har alla redskap du behöver för att bygga vad du vill. Det du vill. Och vet du inte vad du behöver så hjälper vi dig med det med. ',
    offerLead:
      'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. ',
    offerUspOneTitle: 'Utveckling av pilot',
    offerUspOneText:
      'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi.',
    offerUspTwoTitle: 'API',
    offerUspTwoText:
      'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi.',
    offerUspThreeTitle: 'Mobila applikationer',
    offerUspThreeText:
      'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi.',
    offerUspFourTitle: 'AI-lösningar',
    offerUspFourText:
      'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi\n',
    offerIllustrationImage:
      '//images.ctfassets.net/rj4r6yfcesw5/5SAhc8YuL6OaQQsC6miokK/7c09d594e44ffa47781bf1bdb21aae7f/illustrations-group-2.png',
    offerOrder: 1,
    __typename: 'Offer',
  },
]
