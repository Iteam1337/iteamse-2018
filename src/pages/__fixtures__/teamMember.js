export const teamMember = [
  {
    avatar: '',
    email: 'cm@sesame.com',
    gravatar: 'cm.png',
    location: 'New York',
    name: 'Cookie M',
    phoneNumber: '0701234567',
    title: 'Cookie master',
    __typename: 'TeamMember',
  },
]
