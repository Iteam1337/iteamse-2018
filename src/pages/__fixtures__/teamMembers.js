export const teamMembers = [
  {
    avatar: '',
    email: 'cm@sesame.com',
    gravatar: 'cm.png',
    location: 'Stockholm',
    name: 'Cookie M',
    phoneNumber: '0701234567',
    title: 'Cookie master',
    short: 'cmm',
    __typename: 'TeamMember',
  },
  {
    avatar: '',
    email: 'count@sesame.com',
    gravatar: 'count.png',
    location: 'GÃ¶teborg',
    name: 'Count van Count',
    phoneNumber: '0707654321',
    title: '1 ahahah',
    short: 'cvc',
    __typename: 'TeamMember',
  },
]
