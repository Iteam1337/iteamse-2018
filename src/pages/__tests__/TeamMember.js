import React from 'react'
import TeamMember from '../TeamMember'
import { render, wait } from 'react-testing-library'
import MockedQuery from '../../utils/test-utils/MockedQuery'
import { teamMember } from '../__fixtures__/teamMember'
import { teamMembers } from '../__fixtures__/teamMembers'

describe('components/Team', () => {
  const mockedResponse = {
    teamMember,
    teamMembers,
  }

  const match = {
    params: {
      shortName: 'rln',
    },
  }

  it('renders TeamMember', async () => {
    const { container } = render(
      <MockedQuery response={mockedResponse}>
        <TeamMember match={match} />
      </MockedQuery>
    )

    await wait()

    expect(container).toMatchSnapshot()
  })
})
