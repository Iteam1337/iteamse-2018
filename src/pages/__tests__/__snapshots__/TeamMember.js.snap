// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Team renders TeamMember 1`] = `
<div>
  <div
    style="height: 700px;"
    class="lazyload-placeholder"
  />
  <div
    class="s7kn9a1-0-GridColumn__GridColumnClean-lcQAdD cdtVAw"
  >
    <div
      class="s3nsc5b-0-PaddedRow-hWTqUj cqBWXm"
      data-test="breadcrumbs"
    >
      <span>
        Rickard Laurin
      </span>
    </div>
    <div
      class="react-reveal"
      style="opacity: 1;"
    >
      <div
        class="PaddedRow-s3nsc5b-0 cKRLCb"
        data-test="block-"
      >
        <div
          class="Block__Content-jdahsb-0 eJFCrU"
        >
          <div
            class="Block__TitleWrap-jdahsb-1 dkYDLt"
          >
            <h3
              class="H3-s19jy0q4-0 bFoQlU"
            >
              Kontakt
            </h3>
          </div>
          <div
            class="Block__Text-jdahsb-3 bdZkSQ"
          >
            <ul
              class="UnstyledList-s1ntpurm-0 gEDRFE"
            >
              <li>
                <a
                  class="NativeLink-s1qo6rwq-0 jNinQt"
                  href="mailto:rickard.laurin@iteam.se"
                >
                  rickard.laurin@iteam.se
                </a>
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
    <div
      class="react-reveal"
      style="opacity: 1;"
    >
      <div
        class="PaddedRow-s3nsc5b-0 cKRLCb"
        data-test="block-"
      >
        <div
          class="Block__Content-jdahsb-0 eJFCrU"
        >
          <div
            class="Block__TitleWrap-jdahsb-1 dkYDLt"
          >
            <h3
              class="H3-s19jy0q4-0 bFoQlU"
            >
              Varför jobbar du med teknik?
            </h3>
          </div>
          <div
            class="Block__Text-jdahsb-3 bdZkSQ"
          >
            <div
              class="Paragraph__StyledParagraph-s13mwy58-0 kBKPHf"
            >
              <div>
                <p>
                  In the early 90's my family got its first 286, it was love at first sight. In my teen years I played a lot of Delta Force and started developing the clan websites. Until I started at KTH I only did minor hobby projects. There I managed my chapters website and assisted younger students in MATLAB. This is when I realized what I really wanted to work with, building beautiful websites.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="react-reveal"
      style="opacity: 1;"
    >
      <div
        class="PaddedRow-s3nsc5b-0 cKRLCb"
        data-test="block-"
      >
        <div
          class="Block__Content-jdahsb-0 eJFCrU"
        >
          <div
            class="Block__TitleWrap-jdahsb-1 dkYDLt"
          >
            <h3
              class="H3-s19jy0q4-0 bFoQlU"
            >
              Bakgrund
            </h3>
          </div>
          <div
            class="Block__Text-jdahsb-3 bdZkSQ"
          >
            <div
              class="Paragraph__StyledParagraph-s13mwy58-0 kBKPHf"
            >
              <div>
                <p>
                  Materials Science and Engineering at KTH (Royal Institute of Technology). Interaction designer at Nackademin.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="react-reveal"
      style="opacity: 1;"
    >
      <div
        class="PaddedRow-s3nsc5b-0 cKRLCb"
        data-test="block-"
      >
        <div
          class="Block__Content-jdahsb-0 eJFCrU"
        >
          <div
            class="Block__TitleWrap-jdahsb-1 dkYDLt"
          >
            <h3
              class="H3-s19jy0q4-0 bFoQlU"
            >
              Specialistkompetens
            </h3>
          </div>
          <div
            class="Block__Text-jdahsb-3 bdZkSQ"
          >
            <div
              class="Paragraph__StyledParagraph-s13mwy58-0 kBKPHf"
            >
              <div>
                <p>
                  HTML5, CSS and JavaScript.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="s7kn9a1-0-GridColumn__GridColumnClean-lcQAdD jrbaIt"
  >
    <div
      class="TeamMembers__Members-gavpkf-0 kEUlpQ"
    >
      <div
        class="TeamMemberDuo__Colleague-s9kjazo-0 foLTeb"
      >
        <a
          class="PrefetchLink__StyledLink-n3ecxs-0 JPiJu"
          href="/teamet/cmm"
        >
          <div
            style="height: 226px;"
            class="lazyload-placeholder"
          />
        </a>
        <div
          class="TeamMemberDuo__Meta-s9kjazo-1 gzvJxw"
        >
          <div
            class="TeamMemberDuo__MetaSection-s9kjazo-3 XeOZD"
          >
            <div
              class="TeamMemberDuo__Title-s9kjazo-2 dFgPDl"
            >
              Cookie master
            </div>
            <div>
              Stockholm
            </div>
          </div>
          <div
            class="TeamMemberDuo__Title-s9kjazo-2 dFgPDl"
          >
            Cookie M
          </div>
          <div>
            <a
              class="NativeLink-s1qo6rwq-0 jNinQt"
              href="tel:0701234567"
            >
              070-123 45 67
            </a>
          </div>
          <a
            class="NativeLink-s1qo6rwq-0 jNinQt"
            href="mailto:cm@sesame.com"
          >
            cm@sesame.com
          </a>
        </div>
      </div>
      <div
        class="TeamMemberDuo__Colleague-s9kjazo-0 foLTeb"
      >
        <a
          class="PrefetchLink__StyledLink-n3ecxs-0 JPiJu"
          href="/teamet/cvc"
        >
          <div
            style="height: 226px;"
            class="lazyload-placeholder"
          />
        </a>
        <div
          class="TeamMemberDuo__Meta-s9kjazo-1 gzvJxw"
        >
          <div
            class="TeamMemberDuo__MetaSection-s9kjazo-3 XeOZD"
          >
            <div
              class="TeamMemberDuo__Title-s9kjazo-2 dFgPDl"
            >
              1 ahahah
            </div>
            <div>
              Göteborg
            </div>
          </div>
          <div
            class="TeamMemberDuo__Title-s9kjazo-2 dFgPDl"
          >
            Count van Count
          </div>
          <div>
            <a
              class="NativeLink-s1qo6rwq-0 jNinQt"
              href="tel:0707654321"
            >
              070-765 43 21
            </a>
          </div>
          <a
            class="NativeLink-s1qo6rwq-0 jNinQt"
            href="mailto:count@sesame.com"
          >
            count@sesame.com
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
